app/scripts/lib/session.js

```diff
@@ -41,7 +41,34 @@ define([
         // ignore the parse error.
       }
 
-      this.set(values);
+      // Update new values, without overwriting items on the prototype.
+      _.each(values, function (value, key) {
+        if (! Session.prototype.hasOwnProperty(key)) {
```

Comment:
I was thinking about this yesterday. If Session were an actual [Backbone model](http://backbonejs.org/#Model), we wouldn't have to do this unnatural `hasOwnProperty` stuff because models store their data in its own field. It's a bit moot because I want to get rid of Session, and converting to a model is well out of the scope of this PR.